tools/runners/run-all.js

```diff
@@ -163,6 +178,8 @@ class Runner {
 
     if (! self.stopped) {
       buildmessage.enterJob({ title: "starting your app" }, function () {
+        self.errorAppRunner.start();
```

Comment:
any reason why we can't put it into the `otherRunners` array? What is the difference of not doing it?